mayastor/src/bdev/dev/nvme.rs

```diff
@@ -127,14 +127,9 @@ impl CreateDestroy for NVMe {
             let errno = unsafe {
                 bdev_nvme_delete(self.name.clone().into_cstring().as_ptr())
             };
-            async {
-                errno_result_from_i32((), errno).context(
-                    nexus_uri::DestroyBdev {
-                        name: self.name.clone(),
-                    },
-                )
-            }
-            .await
+            errno_result_from_i32((), errno).context(nexus_uri::DestroyBdev {
+                name: self.name.clone(),
+            })
```

Comment:
Can `async` be removed from the function signature now?

Reply:
Hey @paulyoong Good question! It can not. The trait defined the async nature of the function, that it is in fact not async is some minor overhead we simply have to pay.

Reply:
:+1: